src/harness/testcases/WINNF_FT_S_DRG_testcase.py

```diff
@@ -0,0 +1,102 @@
+
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_DRG_1(self):
+    """Valid and correct CBSD ID: two deregistrationRequest objects
+    CBSD sends deregistration request to SAS with two deregistrationRequest
+    objects that contain correct and valid CBSD ID, the response should be
+    SUCCESS.
+    """
+
+    # Register the devices
+    registration_request = []
+    for device_filename in ('device_a.json', 'device_c.json'):
+      device = json.load(
+        open(os.path.join('testcases', 'testdata', device_filename)))
+      self._sas_admin.InjectFccId({'fccId': device['fccId']})
+      self._sas_admin.InjectUserId({'userId': device['userId']})
+      registration_request.append(device)
+    request = {'registrationRequest': registration_request}
+    response = self._sas.Registration(request)['registrationResponse']
+    # Check registration response
+    cbsd_ids = []
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      cbsd_ids.append(resp['cbsdId'])
+    del request, response
+
+    # Deregister the device
+    request = {'deregistrationRequest': [
+        {'cbsdId': cbsd_ids[0]},
+        {'cbsdId': cbsd_ids[1]}]}
+    response = self._sas.Deregistration(request)['deregistrationResponse']
+    # Check the deregistration response
+    for x in range(0, 2):
+      self.assertEqual(response[x]['cbsdId'], cbsd_ids[x])
+      self.assertEqual(response[x]['response']['responseCode'], 0)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_DRG_2(self):
+    """Missing CBSD ID: two objects in the DeregistrationRequest.
+    CBSD sends Deregistration Request with two objects to the SAS, first
+    object has the correct CBSD ID, the second does not have CBSD ID. The
+    response for the first object should be SUCCESS. The response for the
+    second object should be FAIL.
+    """
+
+    # Register the devices
+    registration_request = []
+    for device_filename in ('device_a.json', 'device_c.json'):
+      device = json.load(
+        open(os.path.join('testcases', 'testdata', device_filename)))
+      self._sas_admin.InjectFccId({'fccId': device['fccId']})
+      self._sas_admin.InjectUserId({'userId': device['userId']})
+      registration_request.append(device)
+    request = {'registrationRequest': registration_request}
+    response = self._sas.Registration(request)['registrationResponse']
+    # Check registration response
+    cbsd_ids = []
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      cbsd_ids.append(resp['cbsdId'])
+    del request, response
+
+    # Deregister the device
+    request = {'deregistrationRequest': [
+        {'cbsdId': cbsd_ids[0]},
+        {}]}
+    response = self._sas.Deregistration(request)['deregistrationResponse']
+    # Check the deregistration response
```

Comment:
Let's check that the length of `response` is equal to 2 since the CHECK step says it explicitly.